Xamarin.Forms.Xaml.UnitTests/MSBuild/MSBuildTests.cs

```diff
@@ -67,7 +67,7 @@ public void SetUp ()
 			Directory.CreateDirectory (tempDirectory);
 
 			//We need to copy Xamarin.Forms.targets to the test directory, to reliably import them
-			var xamarinFormsTargets = Path.Combine (testDirectory, "..", "..", "..", ".nuspec", XamarinFormsTargets);
+			var xamarinFormsTargets = Path.Combine (testDirectory, "..", "..", "..", "..", ".nuspec", XamarinFormsTargets);
```

Comment:
FYI I'm not sure if it was just me but I couldn't get these unit tests to pass without this change.